app/(pages)/programs/industry-mentoring-program/page.tsx

```diff
@@ -0,0 +1,89 @@
+import {Box, Stack, AspectRatio, Typography} from '@mui/joy';
+import PageHeaderCard from '@/components/PageHeaderCard';
+import Image from 'next/image';
+import HeadingBodyText from '@/components/HeadingBodyText';
+import ProgramsTitleDescription from '@/components/ProgramsTitleDescription';
+import HorizontalLine from '@/components/HorizontalLine';
+import Quote from '@/components/Quote';
+import { impData } from '@/app/data';
+
+
+function Testimonials() {
```

Comment:
are you able to turn the testimonials into a component for both the IMP program and Protege program?

Reply:
done!